x-pack/plugins/security_solution/server/lib/entity_analytics/risk_score/risk_score_data_client.ts

```diff
@@ -235,6 +266,17 @@ export class RiskScoreDataClient {
       )
       .catch(addError);
 
+    if (namespace !== 'default') {
```

Comment:
why do we need this if? 
Can't we just delete the component template regardless of which namespace?
If we have to pass the name we can just do:
```
{ name: namespace === 'default' ? mappingComponentName : nameSpaceAwareMappingComponentName(namespace) }
```

or even just extract the ternary somewhere so we can treat default and namespaced templates uniformly

Reply:
Yes, i think the if condition is redundant now, and the `ignore : [404]` is probably going to take care for most of the use-cases. Removed the if-condition if the next commit